### PR TITLE
Feat Explore Page

### PR DESCRIPTION
**Pull request by rahulcs754**
- Added Explore pages
- Show all posts

### CODE REVIEW

File: src/frontend/Pages/Explore/Explore.js [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+    <>
+      <Nav />
+      <div className="grid-container ">
+        <SidebarMenu />
+        <div className="middle-part">
+          <div className="flex flex-column width-80 margin-auto ">
+            {isLoading ? <div className="loader"></div> : null}
+
+            {allposts.map((item) => (
+              <Post {...item} key={item._id} />
```
Comment (vaibhav-mahalle) [inline src/frontend/Pages/Explore/Explore.js:28]: liked how you have made the use of components.

File: src/frontend/Components/SidebarMenu/SidebarMenu.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export const SidebarMenu = () => {
             <VscHome size={20} />
             Home
           </li>
-          <li className="list-item">
+          <li className="list-item" onClick={() => navigate("/explore")}>
             <VscRemoteExplorer size={20} />
```
Comment (vaibhav-mahalle) [inline src/frontend/Components/SidebarMenu/SidebarMenu.js:20]: VscRemoteExplorer is a little confusing, you could name it a little better.

**Review by vaibhav-mahalle — APPROVED** [review]
Small pr, suggested some changes please look into them.

**Review by nakulsharma15 — APPROVED** [review]
Everything looks good. Small Pr. The code looks nice and clean. Good to go

File: src/frontend/Components/SidebarMenu/SidebarMenu.js (hunk 2 above)
Comment (rahulcs754) [reply src/frontend/Components/SidebarMenu/SidebarMenu.js:20]: it is icon name by default

File: src/frontend/Pages/Explore/Explore.js (hunk 1 above)
Comment (rahulcs754) [reply src/frontend/Pages/Explore/Explore.js:28]: Thanks

**Pull request merged**